src/components/sections/general/form-sections/sync-input-number-field.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from '@dhis2/prop-types'
+import { InputField } from '@dhis2/ui'
+import { FormSection } from './form-section'
+
+export const SyncInputNumberField = ({ syncElement, handleForm }) => {
+    const { label, syncType } = syncElement
+    return (
+        <FormSection>
+            <InputField
+                inputWidth="100px"
+                label={i18n.t('{{label}}', { label })}
```

Comment:
Calling `i18n.t` here with only a variable doesn't do anything, the text needs to be translated before it is passed to this component.